README.md

```diff
@@ -0,0 +1,73 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books: plain JavaScript with objects
+
+Awesome books is an online book store. It's built with HTML, CSS and mostly JS based on a medium-fidelity wireframe given by Microverse. Adding and removing books from the DOM is made in JavaScript functionality. Also, the books are stored in the local storage so they are retreived when the page reloads.
+
+![screenshot](./awesomebooks.png)
+
+## Built With
+
+- HTML5
+- CSS3
+- Visual Studio Code
+- Git & Github
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repo into your machine. For this, I recommend the following steps:
+  - Open a terminal of your choice (e.g. VS Code integrated terminal)
+  - Select the destination folder (On windows, run cd "_Destination folder_")
+  - Run git clone https://github.com/Kimutai01/Awesome-books.git
+- Open the destination folder in your preferred file explorer
+- Open folder "Awesome-books"
+- Open index.html
+  -Open with the browser
+
+### Prerequisites
+
+Any modern web browser will do.
+
+### Setup
+
+Follow the steps described before.
+
```

Comment:
- [ ] [OPTIONAL] It would be nice to rearrange the prerequisites and set up submenus before the getting started instructions which would be better with a subheading, for instance running the website locally.